features/dapp_develop/cli.go

```diff
@@ -0,0 +1,74 @@
+package dapp_develop
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-cmd/cmd"
+
+	e2e "github.com/stellar/system-test"
+)
+
+// return the fn response as a serialized string
+// uses secret-key and network-passphrase directly on command
+func invokeContractFromCliTool(deployedContractId string, contractName string, functionName string, param1 string, e2eConfig *e2e.E2EConfig) (string, error) {
+	args := []string{
+		"contract",
+		"invoke",
+		"--id", deployedContractId,
+		"--rpc-url", e2eConfig.TargetNetworkRPCURL,
+		"--secret-key", e2eConfig.TargetNetworkSecretKey,
+		"--network-passphrase", e2eConfig.TargetNetworkPassPhrase,
+		"--",
+		functionName,
+	}
+
+	if param1 != "" {
+		args = append(args, param1)
+	}
+
+	envCmd := cmd.NewCmd("soroban", args...)
+
+	status, stdOut, err := e2e.RunCommand(envCmd, e2eConfig)
+
+	if status != 0 || err != nil {
+		return "", fmt.Errorf("soroban cli invoke of example contract %s had error %v, %v", contractName, status, err)
```

Comment:
showing `stdOut` would still be helpful here

Reply:
there is option for stdout to be printed from`RunCommand`, if `VerboseOutput=true`